### PR TITLE
push db and media files

### PR DESCRIPTION
**Pull request by Chive**

### CODE REVIEW

File: aldryn_client/cli.py [hunk 1]
```diff
@@ -148,6 +150,67 @@ def pull_media(obj):
     download_media(obj)
 
 
+@project.group(name='push')
+def project_push():
+    """Push db or files from Aldryn"""
```
Comment (stefanfoulis) [inline aldryn_client/cli.py]: "to Aldryn"

File: aldryn_client/cli.py [hunk 2]
```diff
@@ -148,6 +150,67 @@ def pull_media(obj):
...
+        'WARNING',
+        '=======',
+
+        '\nYou are about to push your local database to the test server on '
+        'Aldryn. This will replace ALL data on the test server with the '
+        'data you are about to push, including (but not limited to):',
+        '  - User Accounts',
+        '  - CMS Pages & Plugins',
+
+        '\nYou will also loose all changes that have been made since you have '
```
Comment (stefanfoulis) [inline aldryn_client/cli.py]: changes that have been made on the server since...

File: aldryn_client/localdev/main.py [hunk 3]
```diff
@@ -199,21 +181,64 @@ def load_database_dump(client, path=None, recreate=False):
...
+
         click.secho('Downloaded media files into {}'.format(path), fg='green')
 
 
+def upload_database(client):
+    project_home = utils.get_project_home()
+    website_id = utils.get_aldryn_project_settings(project_home)['id']
+    dump_filename = 'local_dump.sql'
+    dump_filepath = os.path.join('/app/', dump_filename)
+    archive_path = os.path.join(project_home, 'local_db.tar.gz')
```
Comment (stefanfoulis) [inline aldryn_client/localdev/main.py]: how about putting it in ``/data/local_db.tar.gz``?

File: aldryn_client/localdev/main.py [hunk 4]
```diff
@@ -264,7 +289,7 @@ def develop_package(package, no_rebuild=False):
 
     if not no_rebuild:
```
Comment (stefanfoulis) [inline aldryn_client/localdev/main.py]: ``not no_rebuild`` is not not not nice.

**Comment by stefanfoulis** [comment]
:+1:

File: aldryn_client/localdev/main.py (hunk 3 above)
Comment (Chive) [inline aldryn_client/localdev/main.py]: since it's only temporary and less complicated to handle I think we're fine in ``project_root``

**Pull request merged**